google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1767,23 +1867,72 @@ def post_object path,
         #   post.fields["x-goog-date"] #=> "20200128T000000Z"
         #   post.fields["x-goog-signature"] #=> "4893a0e...cd82"
         #
+        # @example Using the `issuer` and `signer` options:
+        #   require "google/cloud/storage"
+        #   require "google/apis/iamcredentials_v1"
+        #   require "googleauth"
+        #
+        #   # Issuer is the service account email that the Signed URL will be signed with
+        #   # and any permission granted in the Signed URL must be granted to the
+        #   # Google Service Account.
+        #   issuer = "service-account@project-id.iam.gserviceaccount.com"
+        #
+        #   # Create a lambda that accepts the string_to_sign
+        #   signer = lambda do |string_to_sign|
+        #     IAMCredentials = Google::Apis::IamcredentialsV1
+        #     iam_credentials_client = IAMCredentials::IAMCredentialsService.new
+        #
+        #     # Get the environment configured authorization
+        #     scopes =  ["https://www.googleapis.com/auth/cloud-platform"]
+        #     iam_credentials_client.authorization = Google::Auth.get_application_default(scopes)
+        #
+        #     request = {
+        #       "payload": string_to_sign,
+        #     }
+        #     response = iam_credentials_client.sign_service_account_blob(
+        #       "projects/-/serviceAccounts/#{issuer}",
+        #       request,
+        #       {}
+        #     )
+        #     response.signed_blob
+        #   end
+        #
+        #   storage = Google::Cloud::Storage.new
+        #
+        #   bucket = storage.bucket "my-todo-app"
+        #   conditions = [["starts-with", "$acl","public"]]
+        #   post = bucket.generate_signed_post_policy_v4 "avatars/heidi/400x400.png", expires: 10,
```

Comment:
Formatting: I suggest putting the keyword arguments underneath the path argument, to avoid breaking 80 characters in the comment.

Reply:
```
        #   post = bucket.generate_signed_post_policy_v4 "avatars/heidi/400x400.png",
        #                                                expires: 10,
        #                                                conditions: conditions,
        #                                                issuer: issuer,
        #                                                signer: signer
```

Reply:
Avoids parens.